examples/example-column-hidden.html

```diff
@@ -0,0 +1,401 @@
+];
+
+var options = {
+  columnPicker: {
+    columnTitle: "Columns",
+    hideForceFitButton: false,
+    hideSyncResizeButton: false,
+    forceFitTitle: "Force fit columns",
+    syncResizeTitle: "Synchronous resize",
+  },
+  editable: true,
+  enableAddRow: true,
+  enableCellNavigation: true,
+  asyncEditorLoading: true,
+  forceFitColumns: false,
+  topPanelHeight: 25
+};
+
+var sortcol = "title";
+var sortdir = 1;
+var percentCompleteThreshold = 0;
+var searchString = "";
+
+function requiredFieldValidator(value) {
+  if (value == null || value == undefined || !value.length) {
+    return {valid: false, msg: "This is a required field"};
+  }
+  else {
+    return {valid: true, msg: null};
+  }
+}
+
+function myFilter(item, args) {
+  if (item["percentComplete"] < args.percentCompleteThreshold) {
+    return false;
+  }
+
+  if (args.searchString != "" && item["title"].indexOf(args.searchString) == -1) {
+    return false;
+  }
+
+  return true;
+}
+
+function percentCompleteSort(a, b) {
+  return a["percentComplete"] - b["percentComplete"];
+}
+
+function comparer(a, b) {
+  var x = a[sortcol], y = b[sortcol];
+  return (x == y ? 0 : (x > y ? 1 : -1));
+}
+
+function toggleFilterRow() {
+  grid.setTopPanelVisibility(!grid.getOptions().showTopPanel);
+}
+
+let elList = document.querySelectorAll('.grid-header .ui-icon');
+elList.forEach(function(el) {
+  el.classList.add('ui-state-default ui-corner-all');
+  el.addEventListener("mouseover", function(e) { e.target.classList.add('ui-state-hover'); });
+  el.addEventListener("mouseleave", function(e) { e.target.classList.remove('ui-state-hover'); });
+});
+
+//$(".grid-header .ui-icon")
+//        .addClass("ui-state-default ui-corner-all")
+//        .mouseover(function (e) {
+//          $(e.target).addClass("ui-state-hover")
+//        })
+//        .mouseout(function (e) {
+//          $(e.target).removeClass("ui-state-hover")
+//        });
+
+document.addEventListener("DOMContentLoaded", function() {
+  // prepare the data
+  for (var i = 0; i < 50000; i++) {
+    var d = (data[i] = {});
+
+    d["id"] = "id_" + i;
+    d["num"] = i;
+    d["title"] = "Task " + i;
+    d["duration"] = "5 days";
+    d["percentComplete"] = Math.round(Math.random() * 100);
+    d["start"] = "01/01/2009";
+    d["finish"] = "01/05/2009";
+    d["effortDriven"] = (i % 5 == 0);
+  }
+
+
+  dataView = new Slick.Data.DataView({ inlineFilters: true });
+  grid = new Slick.Grid("#myGrid", dataView, columns, options);
+  grid.setSelectionModel(new Slick.RowSelectionModel());
+
+  var pager = new Slick.Controls.Pager(dataView, grid, document.querySelector('#pager'));
+  var columnpicker = new Slick.Controls.ColumnPicker(columns, grid, options);
+
+
+  // move the filter panel defined in a hidden div into grid top panel
+  let el = document.querySelector('#inlineFilterPanel');
+  grid.getTopPanel().append(el);
+  el.style.display = 'block';
+  
+  //$("#inlineFilterPanel")
```

Comment:
you can probably delete this piece of code too, it's meant to add a filter in the preheader section of SlickGrid